plugins/nerdicons.lua

```diff
@@ -0,0 +1,190 @@
+-- mod-version:3
+-- Orginal Author: Jipok
+-- Modified by: techie-guy
+-- Doesn't work well with scaling mode == "ui"
+
+-- This is an extension/modification of the nonicons plugin, https://github.com/lite-xl/lite-xl-plugins/blob/master/plugins/nonicons.lua
+-- If you have any issues or want to suggest any changes, please create an issue and add [nerdicons] in the title
+
+-- Any icon can be searched by it's hex from nerdfonts.com/cheat-sheet
+
+-- How to use:
+-- 1. Download a nerd font that you like from nerdfonts.com
+-- 2. Unzip the zip file that you downloaded.
+-- 3. Choose a font file from the unzipped directory and install it.
+-- 4. Copy the font file to the lite-xl-dir/data/fonts and rename it to "icon-nerd-font.ttf".
```

Comment:
Don't use the data dir for user-installed things. Use the user dir (.config/lite-xl).